### PR TITLE
Added walruses to polar bear story

### PR DESCRIPTION
**Pull request by samwalrus**
Please review this, @casey-collab

### CODE REVIEW

File: english/polar-bear/polar-bear.md [hunk 1]
```diff
@@ -18,8 +18,18 @@ For some reason, you forget your fear and watch. The polar bear eventually stops
...
+walruses back..
+
+
 Now you decide to:
 
 [Take a selfie](../selfie/selfie.md)
 
 [Do a dance](../dance/dance.md)
+
+[and ride through the worm whole](../worm_hole/worm_hole.md)
```
Comment (casey-collab) [inline english/polar-bear/polar-bear.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/polar-bear/worm_hole/worm_hole.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+In the new dimension, millions of walruses with kaleidoscope eyes, turn and follow
```
Comment (casey-collab) [inline english/polar-bear/worm_hole/worm_hole.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**